tests/e2e/binding_modes_with_topology.go

```diff
@@ -80,14 +76,16 @@ var _ = ginkgo.Describe("[csi-topology-vanilla] Topology-Aware-Provisioning-With
 	})
 	verifyTopologyAwareProvisioning := func(f *framework.Framework, client clientset.Interface, namespace string, scParameters map[string]string,
 		allowedTopologies []v1.TopologySelectorLabelRequirement) {
+		var cancel context.CancelFunc
```

Comment:
what was the issue with the global context?

Reply:
It was throwing this error :  Unexpected error:
   <*errors.errorString | 0xc000096250>: {
         s: "context canceled",
      }      context canceled